package.json

```diff
@@ -1,29 +1,82 @@
 {
-  "name": "my-project",
-  "version": "0.1.0",
+  "name":"WeeUrl",
+    "version": "1.0.2",
+    "description": "Reliable Development Service At Scale",
+    "keywords": [
+        "typescript",
+        "react",
+        "next",
+        "eslint",
+        "prettier",
+        "jest",
+        "google",
+        "google-analytics"
+    ],
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
-    "start": "next start",
-    "lint": "next lint"
+    "build": " next build",
+    "start": "next start"
   },
   "dependencies": {
-    "@radix-ui/react-popover": "1.0.2",
-    "@types/node": "18.11.10",
-    "@types/react": "18.0.26",
-    "@types/react-dom": "18.0.9",
-    "eslint": "8.29.0",
-    "eslint-config-next": "13.0.6",
-    "next": "13.0.6",
+    "@emailjs/browser": "^3.10.0",
+    "@fontsource/karla": "^4.5.10",
+    "@fontsource/sora": "^4.5.10",
+    "@headlessui/react": "^1.7.4",
+    "@heroicons/react": "^2.0.13",
+    "@radix-ui/react-accordion": "^1.0.1",
+    "@radix-ui/react-alert-dialog": "^1.0.2",
+    "@radix-ui/react-aspect-ratio": "^1.0.1",
+    "@radix-ui/react-avatar": "^1.0.1",
+    "@radix-ui/react-checkbox": "^1.0.1",
+    "@radix-ui/react-collapsible": "^1.0.1",
+    "@radix-ui/react-context-menu": "^2.0.1",
+    "@radix-ui/react-dialog": "^1.0.2",
+    "@radix-ui/react-dropdown-menu": "^2.0.1",
+    "@radix-ui/react-hover-card": "^1.0.2",
+    "@radix-ui/react-icons": "^1.1.1",
+    "@radix-ui/react-label": "^2.0.0",
+    "@radix-ui/react-navigation-menu": "^1.1.1",
+    "@radix-ui/react-popover": "^1.0.2",
+    "@radix-ui/react-progress": "^1.0.1",
+    "@radix-ui/react-radio-group": "^1.1.0",
+    "@radix-ui/react-scroll-area": "^1.0.1",
+    "@radix-ui/react-select": "^1.1.1",
+    "@radix-ui/react-separator": "^1.0.1",
+    "@radix-ui/react-slider": "^1.1.0",
+    "@radix-ui/react-switch": "^1.0.1",
+    "@radix-ui/react-tabs": "^1.0.1",
+    "@radix-ui/react-toast": "^1.1.1",
+    "@radix-ui/react-toggle": "^1.0.1",
+    "@radix-ui/react-toggle-group": "^1.0.1",
+    "@radix-ui/react-toolbar": "^1.0.1",
+    "@radix-ui/react-tooltip": "^1.0.2",
+    "@testing-library/react": "^13.4.0",
+    "@types/jest": "^29.2.6",
+    "@vercel/analytics": "^0.1.5",
+    "classnames": "^2.3.2",
+    "next": "13.0.3",
+    "next-images": "^1.8.4",
+    "next-seo": "^5.14.1",
     "next-themes": "^0.2.1",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
-    "typescript": "4.9.3"
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-github-btn": "^1.4.0",
```

Comment:
I don't see this package is used anywhere in this PR. However, in my view, we shouldn't use this type of package instead we can create our own button element.